types/pdfkit/index.d.ts

```diff
@@ -14,23 +14,32 @@
 
 declare namespace PDFKit {
     interface PDFGradient {
-        new (document: any): PDFGradient;
+        new(document: any): PDFGradient;
         stop(pos: number, color?: string | PDFKit.PDFGradient, opacity?: number): PDFGradient;
         embed(): void;
         apply(): void;
     }
 
     interface PDFLinearGradient extends PDFGradient {
-        new (document: any, x1: number, y1: number, x2: number, y2: number): PDFLinearGradient;
+        new(document: any, x1: number, y1: number, x2: number, y2: number): PDFLinearGradient;
         shader(fn: () => any): any;
         opacityGradient(): PDFLinearGradient;
     }
 
     interface PDFRadialGradient extends PDFGradient {
-        new (document: any, x1: number, y1: number, x2: number, y2: number): PDFRadialGradient;
+        new(document: any, x1: number, y1: number, x2: number, y2: number): PDFRadialGradient;
         shader(fn: () => any): any;
         opacityGradient(): PDFRadialGradient;
     }
+
+    interface PDFTilingPattern {
+        new(document: any): PDFTilingPattern;
```

Comment:
We should stay in line with the other definitions and add the missing constructor arguments (bBox, xStep, yStep, stream)